index.css

```diff
@@ -0,0 +1,81 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+:root {
+  --primary: #6070ff;
+  --secondary: #344563;
+  --primary-light: #7f8cff;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+}
+
+@media screen and (max-width: 480px) {
+  .landingPage {
+    width: 100%;
+    height: 100vh;
+    background-image: url('./images/mobile-bg.png');
+    background-position: center;
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
```

Comment:
- [ ] [REQUIRED] Good job so far! In my opinion, you have too much head space at the top of your `header` elements, please reduce it like the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=1gjr7tWmuQCR6lVD-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/211911719-a0334776-f5b3-45e8-8f72-f9726ac31d57.png)